test/lib/jdk/test/lib/Platform.java

```diff
@@ -260,6 +260,34 @@ public static boolean hasSA() {
         return true;
     }
 
+
+    public static boolean hasPlistEntriesOSX() throws IOException {
+        // Find the path to the java binary.
+        String jdkPath = System.getProperty("java.home");
+        Path javaPath = Paths.get(jdkPath + "/bin/java");
+        String javaFileName = javaPath.toAbsolutePath().toString();
+        if (Files.notExists(javaPath)) {
+            throw new FileNotFoundException("Could not find file " + javaFileName);
+        }
+
+        // Run codesign on the java binary.
+        ProcessBuilder pb = new ProcessBuilder("codesign", "--display", "--verbose", javaFileName);
+        pb.redirectErrorStream(true); // redirect stderr to stdout
+        Process codesignProcess = pb.start();
+        BufferedReader is = new BufferedReader(new InputStreamReader(codesignProcess.getInputStream()));
```

Comment:
Maybe do the BufferedReader stuff in a try-with-resources and then return false instead of potentially throwing an IOException?